### PR TITLE
COOPR-634: Add /resolve API for resolving templates with extend (parent) and includes (partials)

### PR DESCRIPTION
**Pull request by rpedanov**

### CODE REVIEW

**Comment by rpedanov** [comment]
@jwang47 please check it.

**Comment by rpedanov** [comment]
I think build should be restarted

File: coopr-server/src/main/java/co/cask/coopr/http/handler/AdminHandler.java [hunk 1]
```diff
@@ -587,7 +593,8 @@ public void getProviders(HttpRequest request, HttpResponder responder) {
 
     try {
       responder.sendJson(HttpResponseStatus.OK, entityStoreService.getView(account).getAllProviders(),
-                         new TypeToken<Collection<Provider>>() { }.getType(), gson);
+                         new TypeToken<Collection<Provider>>() {
+                         }.getType(), gson);
```
Comment (jwang47) [inline coopr-server/src/main/java/co/cask/coopr/http/handler/AdminHandler.java]: Put this on same line as previous line.

**Comment by jwang47** [comment]
One minor comment. Please run unit tests locally. If they pass, then we can merge after you fix the comment.

**Comment by wolf31o2** [comment]
@jwang47 only people with commit rights to the repo can restart the build... I just restarted it.

**Comment by rpedanov** [comment]
@jwang47 please check it

**Comment by jwang47** [comment]
LGTM

**Pull request merged**